src/Shared/Util/dispatchOpen.test.ts

```diff
@@ -40,7 +41,8 @@ describe('dispatchOpen()', () => {
             const openSpy = spy(actions, 'open');
 
             const mockDetectBodyCollision = () => ({
-                maxVisibleItemsOverride: 3
+                maxVisibleItemsOverride: 3,
+                type: CollisionType.TOP
```

Comment:
This seemed to be required when i ran the tests but tbh i dont have a lot of context here